### ISSUE TITLE
anti-HLA antibodies

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7100]
**Preferred term label:**
Anti-HLA antibodies

**Synonyms**
Anti-human leukocyte antigens (HLA) antibodies 

**Definition (free text, please give PubMed ID)**
Presence of anti-HLA antibodies
29985463 [PMID] _23628391 [PMID] _33709433 [PMID] _ 28757117 [PMID]

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Anti-HLA hyperimmunization ORPHA 2194

**Comment by pnrobinson** [comment]
added as HP:0034016
